src/bin/common/solving_loop.ml

```diff
@@ -682,61 +767,79 @@ let main () =
       unsupported_opt name; st
   in
 
-  let handle_optimize_stmt ~is_max loc id (term : DStd.Expr.Term.t) st =
-    let contents = `Optimize (term, is_max) in
-    let stmt = { Typer_Pipe.id; contents; loc; attrs = []; implicit = false } in
-    let cnf =
-      D_cnf.make (State.get State.logic_file st).loc
-        (State.get solver_ctx_key st).ctx stmt
-    in
-    (* Using both optimization and incremental mode may be wrong if
-       some optimization constraints aren't at the toplevel.
-       See issue: https://github.com/OCamlPro/alt-ergo/issues/993. *)
```

Comment:
Please don't remove comments pointing to issues that are not solved.

Reply:
This still applies, please do not remove comments pointing to issues that are not solved.